funnel/forms/profile.py

```diff
@@ -50,8 +50,11 @@ def set_queries(self):
 @Profile.forms('logo')
 class ProfileLogoForm(forms.Form):
     logo_url = forms.URLField(
-        __("Logo image URL"),
-        description=__("URL for profile logo image"),
+        __("Logo image URL from Imgee"),
```

Comment:
We're not using the "Imgee" brand in public. This should be just "Logo image URL"